classes/rubygems.bbclass

```diff
@@ -5,7 +5,10 @@
 
 GEMLIB_VERSION = "3.2.0"
 
-GEM_SRC ?= "http://rubygems.org/gems"
+GEM_SRC ?= "https://rubygems.org/gems"
+
+# Enforce IPv4 connection - IPv6 seems to be broken
+FETCHCMD_wget ?= "/usr/bin/env wget -t 2 -T 30 --passive-ftp -4"
 
 GEM_NAME ?= ""
 GEM_VERSION ?= "${PV}"
```

Comment:
potential issue: 'FETCHCMD' is set specific to ['wget'], but isn't known from PACKAGES, MACHINE, DISTRO or resources